suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -117,7 +117,8 @@ const discoverAccountsByDescriptorThunk = createThunk(
                 coin: bundleItem.coin,
                 descriptor: bundleItem.descriptor,
                 useEmptyPassphrase: true,
-                details: 'txs',
+                details: 'basic',
```

Comment:
Should be set to "tokens" at least. Without it the ETH token details won't be fetched.

Reply:
Also the `txs` is crucial for discovery of ADA accounts. IF we fetch ADA account with a `basic` details, then the account receive addresses are not downloaded. This results in showing ADA xpub instead of the receive address during the receive flow.

Reply:
solved in https://github.com/trezor/trezor-suite/pull/10358/files